<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>525</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit{
background: transparent;
border: none;
border-bottom: 1px solid rgba(0, 0, 0, 155);
font-family: sans-serif;
}
QMainWindow{
background: rgb(0, 80, 106);
font-family: sans-serif;
}

QGroupBox{
background: white;
border: 1px solid rgba(255, 255, 255, 200);
font-family: sans-serif;
border-radius:5px;
}
QGroupBox#statusBox{
border-radius: 0px;
}
QPushButton#exitButton, QPushButton#minimizeButton{
border-radius: 0px;
border: none;
background: transparent;
}
QPushButton#exitButton:hover{
background: rgb(255, 46, 46);
}
QPushButton#minimizeButton:hover{
background: rgb(223, 223, 223);
}
QPushButton:hover{
background: rgb(75, 195, 220);
border: none;
}

QPushButton{
background: rgb(58, 180, 208);
border-radius: 10px;
border: 1px solid rgb(18, 140, 168);
font-family: sans-serif;
}

QTableView, QListView{
background: transparent;
font-family: sans-serif;
height: 111px;
border-radius:5px;
outline: 0;
spacing: 10px;
}

QAbstractItemView{
background:transparent;
outline: 0;

}

QAbstractItemView:item{
background: rgb(58, 180, 208);
border-radius: 5px;
padding:5px;
outline: 0;
color: black;
margin: 2px;

}

QAbstractItemView:item:hover{
background: rgb(75, 195, 220);
}

QAbstractItemView:item:selected{
background: rgb(75, 195, 220);
}
QTableCornerButton::section{
background: transparent;
}
QProgressBar {
    border: 2px solid grey;
    border-radius: 5px;
    text-align: center;
}

QProgressBar::chunk {
    background-color: #05B8CC;
    width: 20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="statusBox">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>30</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>5</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Проект</string>
     </property>
    </widget>
    <widget class="QPushButton" name="minimizeButton">
     <property name="geometry">
      <rect>
       <x>740</x>
       <y>0</y>
       <width>30</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>—</string>
     </property>
    </widget>
    <widget class="QPushButton" name="exitButton">
     <property name="geometry">
      <rect>
       <x>770</x>
       <y>0</y>
       <width>30</width>
       <height>27</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
      </font>
     </property>
     <property name="text">
      <string>X</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>761</width>
      <height>451</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>sans-serif</family>
     </font>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Название курса</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>411</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>sans-serif</family>
       <pointsize>13</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="placeholderText">
      <string>Описание курса</string>
     </property>
    </widget>
    <widget class="QPushButton" name="createButton">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>370</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Создать курс</string>
     </property>
    </widget>
    <widget class="QListWidget" name="mList">
     <property name="geometry">
      <rect>
       <x>475</x>
       <y>30</y>
       <width>241</width>
       <height>331</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Добавить тему...</string>
      </property>
     </item>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>400</y>
       <width>761</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Georgia</family>
      </font>
     </property>
     <property name="text">
      <string>*Это демонстрационная версия, в полной
версии доступно создание курсов.</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>410</y>
       <width>131</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>&lt;-   Назад</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
